src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -579,4 +611,12 @@ TransportNetwork getTransportNetwork(String idString) {
 	private String getString(int res) {
 		return context.getString(res);
 	}
+
+	private String region(@StringRes int name, String flag) {
+		return flag + " " + getString(name);
```

Comment:
Yes, there's several possibilities for how to do this. The flag doesn't need to be part of the same string. It could be two strings that are combined on view creation in the adapter. But that's out of scope of this MR. Feel free to work on this afterwards.